tests/habit.test.js

```diff
@@ -331,13 +331,31 @@ describe('THE HABITS TEST SUITE', () => {
       done();
     });
 
-    it('Should allow habit name update when name does not already exist', async done => {
+    it('Should allow habit name update when name does not already exist and habitActive is boolean', async done => {
       const response = await request
         .patch(`${baseHabitRoute}/user/${adminId}/${adminHabitId}`)
         .set({ Authorization: adminToken })
-        .send({ name: 'Stoop to conquer' });
+        .send({ name: 'Stoop to conquer', habitActive: true });
```

Comment:
Since the habits are set to true by default, the test here would probably be best made `false`.